package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "@top-gg/sdk",
-  "version": "3.1.1",
+  "version": "3.1.2",
```

Comment:
There's also no need to bump a change that only makes a difference to people who clone from GitHub.

Reply:
ok